src/utils/mustache.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as chrono from 'chrono-node';
+
+export const replaceMustacheTemplates = (line: string, epoch = true) => {
+  return line.replace(/\{\{(.*?)\}\}/g, (match: string, p1: string) => {
+    try {
+      const parsedDate = chrono.parseDate(p1);
+      if (epoch) {
+        return parsedDate?.getTime().toString() as string;
+      }
+      return parsedDate?.toDateString() as string;
```

Comment:
```suggestion
      if (!parsedDate) throw new Error('Date is null');
      if (epoch) {
        return parsedDate.getTime().toString();
      }
      return parsedDate.toDateString();
```

Reply:
Updated it [here](https://github.com/opensearch-project/skills-eval/pull/15/files#diff-43e12eaa7a351d06b82fa867f1c6bc86a0a39dbd61ece5054a22efc175a5a266R10-R15)
```
    const parsedDate = chrono.parseDate(p1);
    if (!parsedDate) return match;
    if (epoch) {
      return parsedDate.getTime().toString();
    }
    return parsedDate.toDateString();
```
Removed the `try... catch` because there are other test cases that may use double curly brackets, like [here](https://github.com/opensearch-project/skills-eval/blob/main/data/indices/alerting/.opendistro-alerting-config/documents.ndjson). Is there a better way to capture these natural language date strings? We could also check for a different wrapper, such as triple curly brackets.

Reply:
got it. if you want you can check and make sure at least one keyword like `day`, `hour`, etc. exists, or invent some new syntax like `${{ xxx }}` instead of `{{ xxx }}`, but i think it's ok if it fails to parse the string then it will leave the string as is

Reply:
In fb3f182, I've updated the syntax from `{{ xxx }}` to `${{ xxx }}` and kept the behavior of leaving the string as if the parser fails, just in case it may conflict with other documents that require using `{{ xxx }}`.